src/policy/fees.h

```diff
@@ -267,6 +279,10 @@ class CBlockPolicyEstimator
     unsigned int HistoricalBlockSpan() const EXCLUSIVE_LOCKS_REQUIRED(m_cs_fee_estimator);
     /** Calculation of highest target that reasonable estimate can be provided for */
     unsigned int MaxUsableEstimate() const EXCLUSIVE_LOCKS_REQUIRED(m_cs_fee_estimator);
+
+    /** A non-thread-safe helper for the removeTx function */
+    bool removeTxNonThreadSafeHelper(const uint256& hash, bool inBlock)
```

Comment:
```suggestion
    bool _removeTx(const uint256& hash, bool inBlock)
```

According to similar methods:

```cpp
    void _RelayTransaction(const uint256& txid, const uint256& wtxid)
        EXCLUSIVE_LOCKS_REQUIRED(cs_main);
```

Reply:
Thanks! It's great to have a common approach for such a naming case. [Updated](https://github.com/bitcoin/bitcoin/pull/22014#issuecomment-845784457).